types/powerbi-visuals-tools/tslint.json

```diff
@@ -0,0 +1,14 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
```

Comment:
I don't even see violations for some of these disabled rules (unused generics?)

I'd much prefer that they all remain enabled and you add per-line exclusions with justifications.